dts/bindings/display/sitronix,st7789v.yaml

```diff
@@ -11,7 +11,7 @@ include: spi-device.yaml
 properties:
     reset-gpios:
       type: phandle-array
-      required: true
+      required: false
```

Comment:
nit: removing required is equivalent to false

Reply:
Personally I like that is was explicitly optional and other properties also have `required: false`. Do you want me to remove all `required: false` from this file?